src/praisonai-agents/test_tool_retry.py

```
@@ -0,0 +1,84 @@
+    try:
+        # Test config import
+        from praisonaiagents import ToolRetryConfig
+        config = ToolRetryConfig()
+        
+        # Test OnRetryInput import  
+        from praisonaiagents.hooks import OnRetryInput
+        retry_input = OnRetryInput()
+        
+        print("✅ All imports successful!")
+        return True
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)